src/services/mail.service.ts

```diff
@@ -1,11 +1,8 @@
-import Token from "../sequelize/models/Token";
-import { IUser, SUBJECTS } from "../types";
+import { IUser } from "../types";
 import { env } from "../utils/env";
-import { generateMagicLinkToken } from "../utils/jsonwebtoken";
 import transporter from "../utils/transporter";
-import { verifyOtpTemplate } from "../email-templates/verifyotp";
 
-export const sendEmailService = async (user: IUser, subject: string, template: any, token: number) => {
+export const sendEmailService = async (user: IUser, subject: string, template: any) => {
```

Comment:
do you think this function can be re-used , why that user can you put email instead whole user object

Reply:
yes its reusable i removed token since token doesn't have naything todo with that sendEmailService, token or any other related info has be passed as argument in emailTemplate not in service

Reply:
also pass email to send to not whole user